server.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import socket
+
+addr = ("127.0.0.1", 8000)
+
+
+def setup():
+    """
+    Create new socket, and bind localhost to socket.
+    Set socket to listen, and return socket information.
+    """
+    sock = socket.socket(
+        socket.AF_INET, socket.SOCK_STREAM, socket.IPPROTO_IP
+    )
+    sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+    sock.bind(addr)
+    sock.listen(1)
+    return sock
+
+
+def response_ok():
+    """Return Header and Body information for Response 200."""
+    response_headers = 'HTTP/1.1 200 OK\nContent-Type: text/html'
+    response_body = '<html><body><h1></h1>'\
+        '<p>All good here, captain.</p></body></html>'
+
+    return response_headers + response_body
+
+
+def response_error():
+    """Return Header and Body information for Response 500."""
+    response_headers = 'HTTP/1.1 500 Internal Server Error\n' +\
+        'Content-Type: text/html'
```

Comment:
put in parens around these two strings and use the "abutting strings concatenate" feature.  This is hard to read.  The line-continuation character should be a tool of last resort.